app/views/home/index.html.erb

```diff
@@ -6,12 +6,40 @@
   </p>
 </div>
 
-<h2><%= t('latest_projects') %></h2>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-4">
+      <h2><%= t('latest_projects') %></h2>
 
-<ul>
-  <% cache do %>
-    <% @latest_projects.each do |project| %>
-      <li><%= link_to(project.name, project_path(project.id)) %></li>
-    <% end %>
-  <% end %>
-</ul>
+      <ul>
+        <% cache do %>
```

Comment:
I think @danielkza is right.
From the [Rails' guides on fragment caching](http://guides.rubyonrails.org/caching_with_rails.html#fragment-caching):

> The cache block in our example will bind to the action that called it and is written out to the same place as the Action Cache, which means that if you want to cache multiple fragments per action, you should provide an action_suffix to the cache call

For a single cache fragment we were ok, but with multiple cache fragments we need to, at least, use the `action_suffix`.